geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/AcceptorImpl.java

```diff
@@ -971,6 +981,7 @@ private void checkForStuckKeys() {
   public void runSelectorLoop() {
     // int zeroEventsCount = 0;
     try {
+      long lastCheckedTime = System.currentTimeMillis();
```

Comment:
Since this is all in the same thread you should use the nano clock (System.nanoTime) and not the millisecond clock.  Millisecond clock probes are slower, especially on a virtual machine, and we don't want the selector loop to be unnecessarily delayed by reading the clock.

Reply:
requested change was done